docs/_data/topnav.yml

```
@@ -21,7 +21,7 @@ topnav_dropdowns:
     - title: Contact
       folderitems:
 
-        - title: Slack
+        - title: Slack (#acre2 channel)
```

```suggestion
        - title: Slack `#acre2`
```
Can you check if this works? :thinking:

---

Disregard, doesn0t.